bangazon-clumsy-penguins/BangazonAPI
# Eh sql file

# Name - Created SQL files

## Description
##### Provide clarity to any potentially complex code:
Tables should be added according to the order they appear on the ERD to avoid any dependency conflicts. I grouped the scripts to add and delete the tables according to whether they were Company or Customer tables.

##### Explain reasons behind organizational or architectual decisions you made:
The code to add and delete is separated into two files so that they can be run and tested separately without needing to comment out any code.

## Steps to Test
1. Create a database file named BangazonAPI.db
1. Run the SQL scripts in bangazon.sql to verify that all 13 tables on the ERD are created
1. Run the SQL scripts in delete-bangazon.sql to verify that all tables can be deleted correctly

##### System configuration.
None

##### 3rd party libraries that need to be installed.
None

##### Command line utilities to run.
None

##### Link to Feature Ticket
#19
